shetland/core/src/main/java/org/n52/shetland/ogc/swe/SweDataStream.java

```diff
@@ -29,22 +29,10 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
-public class SweDataStream implements HasDefaultEncoding<SweDataStream>, Copyable<SweDataStream> {
 
-    /**
-     * optional: swe:description[0..1]
-     */
-    private String description;
-
-    /**
-     * optional: swe:label [0..1]
-     */
-    private String label;
+public class SweDataStream extends AbstractSWEIdentifiable
```

Comment:
You can use `SweAbstractDataComponent `instead of creating a new `AbstractSWEIdentifiable`

Reply:
If i take a look at https://portal.ogc.org/files/?artifact_id=41157 Section 7.2.2. it's the other way around - `SweAbstractDataComponent` should not define those field itself but extends `AbstractSWEIdentifiable`. Or do i miss something here?

Reply:
From the spec perspective you are correct but we have _merged_ some classes from the spec into one class in our implementation